.travis.yml

```
@@ -139,6 +139,24 @@ matrix:
       - tools/setup_android.sh
      script:
       - tools/build_android.sh --release --verbose
+   - name: "Cross build x86 from x86_64"
```

Better shorten this, otherwise it won't fit in the Travis CI UI.
```suggestion
   - name: "x86 cross-build"
```